tests/bsim/bluetooth/host/privacy/device/src/test_undirected_peripheral.c

```diff
@@ -233,13 +233,9 @@ static void update_adv_params(struct bt_le_ext_adv *adv, enum adv_param_t adv_pa
 
 	if (use_ext_adv && adv_params == NCONN_SCAN) {
 		uint8_t data[4] = {0x61, 0x6c, 0x65, 0x64};
-		struct bt_data sd;
+		struct bt_data sd = BT_DATA(BT_DATA_NAME_COMPLETE, data, ARRAY_SIZE(data));
```

Comment:
This is fine, but alternatively you could also do 
```suggestion
		struct bt_data sd = BT_DATA_BYTES(BT_DATA_NAME_COMPLETE, data);
```

I would also consider using a string or `char`s in the above array (looks like it's "aled") for increased readability

Reply:
Thanks, I updated to follow your recommendation and changed the value 😅